core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -3198,7 +3196,7 @@ private void setBuildsAndWorkspacesDir() throws IOException, InvalidBuildsDir {
      * Checks the correctness of the newBuildsDirValue for use as {@link #buildsDir}.
      * @param newBuildsDirValue the candidate newBuildsDirValue for updating {@link #buildsDir}.
      */
-    @VisibleForTesting
+    //@VisibleForTesting
```

Comment:
This is in the `com.google.common.annotations` package, not the `com.google.common.collect` package, so it's technically out of the scope of this PR (which focuses on the `com.google.common.collect` package).

I don't feel strongly one way or another about removing it or keeping it. If we keep it, we could potentially use it with Checkstyle's [`VisibilityModifier`](https://checkstyle.sourceforge.io/config_design.html#VisibilityModifier) check in the future. I am working on a Checkstyle PR for this repository and I may look into this.

However, I do feel strongly that we remain consistent. This PR is inconsistent because it comments out `//@VisibleForTesting` in some cases but leaves it uncommented in other cases.

Overall, I think it would be best to revert the `@VisibleForTesting` changes in this PR and to open a separate PR to cover `com.google.common.annotations.VisibleForTesting` by either keeping it and taking advantage of it through static analyzers or else removing it consistently throughout the whole codebase.

Reply:
sorry been to far enthusiast here 😄